.python-version

```diff
@@ -0,0 +1 @@
+2.7.15
```

Comment:
Could you add this file to .gitignore and remove it?

Reply:
Whoops, yes, sorry!

Reply:
As a side-note, it might be good to add to our [.gitignore file in the open-source-project-template](https://github.com/cfpb/open-source-project-template/pull/30).

Reply:
Maybe. It should really be in my `.gitignore_global`.